gateway/api/migrations/0014_runtimejob.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.2 on 2024-01-20 01:44
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
Do we need runtime job in this PR?

Reply:
@IceKhan13 No, I can take it out.

Reply:
We can leave it, if you think it is ready to be reviewed.
I only saw public in title and description of this PR that is why I want to make sure it is intended :)

Reply:
if we want to leave it. We can just add it into description and crosslink issues.

Reply:
Ah, we have runtime model here https://github.com/Qiskit-Extensions/quantum-serverless/pull/1177

Reply:
Interesting that we still have this file even though it should be in while we are looking for best configuration

Reply:
ah, files called differently. We should take this one out

Reply:
hmm, nope files called the same way 🤔